.gitignore

```diff
@@ -105,6 +106,7 @@ Podfile.lock
 
 # Other ignored files
 google-services.json
+!sample_app/android/app/google-services.json
```

Comment:
Can we move this to the exceptions list in the end.

Reply:
sure 👍